app/gosns/gosns_test.go

```diff
@@ -92,6 +58,17 @@ func TestPublishhandler_POST_SendMessage(t *testing.T) {
 	form.Add("Message", "TestMessage1")
 	req.PostForm = form
 
+	// Prepare existant topic
+	topic := &app.Topic{
+		Name: "UnitTestTopic1",
+		Arn:  "arn:aws:sns:local:000000000000:UnitTestTopic1",
+	}
+	app.SyncTopics.Topics["UnitTestTopic1"] = topic
+	defer func() {
+		// Clear topic
+		delete(app.SyncTopics.Topics, "UnitTestTopic1")
```

Comment:
Can we use `utils.ResetApp()` in these cases instead?  I built it so it would wipe out all the queues and topics completely for each test, so we can start fresh each time.

Reply:
Applied!